index.js

```diff
@@ -108,12 +117,13 @@ exports.login = async function (options) {
             input: process.stdin,
             output: process.stdout
         });
+        const redirectAutomaticallyHint = tempUrl ? '(if not redirected automatically)' : '';
         const getAccountFromConsole = async () => {
             return await new Promise((resolve) => {
                 rl.question(
                     chalk`Please authorize at least one account at the URL above.\n\n` +
                     chalk`Which account did you authorize for use with NEAR Shell?\n` +
-                    chalk`{bold Enter it here (if not redirected automatically):}\n`, async (accountId) => {
+                    chalk`{bold Enter it here${redirectAutomaticallyHint}:}\n`, async (accountId) => {
```

Comment:
better to add whitespace after the `e` in `"here"`

Reply:
otherwise it looks cramped
![image](https://user-images.githubusercontent.com/24913/80782007-253aba00-8b7d-11ea-99e5-c14677729d1a.png)